docs/DeveloperGuide.md

```diff
@@ -851,7 +851,9 @@ Team size: 5
 1. **Standardise uppercase for NRIC displayed in the person card:** If using lowercase NRIC when adding elderly: `add i/s5305394G n/John Doe p/98765432 e/johnd@example.com a/311, Clementi Ave 2, #02-25 c/7`, it is displayed as `s5305394G` in the person card (no change in case). We plan to standardise the displayed NRIC to be in uppercase in the person card for easier reading for users: `S5305394G`.
 2. **Relax constraint for name field:** The current constraint for the name field is alphanumeric characters and spaces which could be too restrictive if users want to add elderly with names that include other special characters (e.g. `João da Silva` or `Arjun Singh s/o Vijay Singh`). We plan to relax this constraint so that other characters that are commonly used in names are allowed. A non-exhaustive list of characters that will be allowed after this enhancement consists of diacritics (e.g. `ç`, `ñ`, `é`, `ü`), punctuation and symbols (e.g. `-`, `'`, `.`, `/`),  extended Latin characters (e.g. `ø`, `å`, `ð`), Chinese characters (e.g. `陈`, `小`, `明`) and Tamil characters (e.g.`ர`, `ஜ`).
 3. **Allow modification to elderly in the history view:** Currently, users can view the call history and profile view of the elderly in the history view together. This allows users to view the details of the elderly, which may give the wrong impression that they can make modifications to the elderly such as performing mark command using NRIC in the history view (e.g. `mark SXXXXXXXH`). However, users are unable to perform any modification to elderly in the history view. We plan to allow users to make modifications to the elderly whose details are specified in the history view such as making changes to the elderly details and marking the elderly as called, etc.
-4. **Maintain the filtered person list view after modification:** Currently, if a user uses the `find` command to filter the person list, and then proceeds to modify the person list (with `add`, `edit`, `mark`  etc.), the person list will reset to show all entries. We plan to change this behaviour such that the current filtered person list will stay filtered (with the same filter as applied by the `find` command previously), even after modifications.
+4. **Improve error messages:** Currently, the error messages inform users that a command is incorrect. We plan to improve the error messages to provide more specific feedback to users on what went wrong. For example, if a user has missing/extra fields in the `add` command, the error message will specify which fields are causing the error.
```

Comment:
I think this has to be more specific, if not I think? they can flag it as a documentation bug. Like the example you gave about ` the error message will specify which fields are causing the error`, should replace the slightly vague enhancement here `We plan to improve the error messages to provide more specific feedback to users on what went wrong`

just a thought, but not exactly sure what the correct move is

Reply:
Could you elaborate on this?